### PR TITLE
fix: fix shaders 404 when base_url is /

### PR DESCRIPTION
**Pull request by thkruz**
Could not get it to load with / as my base_url (took a bit to figure that out). I assume you aren't doing that locally.

### CODE REVIEW

File: src/viewer/ShaderStore.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ShaderStore {
   shaderData: Record<string, string> = {};
 
   constructor (appBaseUrl = '') {
-    this.baseUrl = `${appBaseUrl}${this.baseUrl}`;
+    this.baseUrl = `${appBaseUrl}${this.baseUrl}`.replace(/\/\//g, '/');
```
Comment (ajmas) [inline src/viewer/ShaderStore.ts:21]: Would this not break any URLs that may start with `https://` ?

Comment (ajmas) [inline src/viewer/ShaderStore.ts:21]: Would this not break any URLs that may start with `https://` ?

**Comment by ajmas** [comment]
During the update of the dependencies I ran into this issue, but solved the problem differently. See:

https://github.com/ajmas/StuffInSpace/pull/62

**Comment by thkruz** [comment]
Agreed. Poorly thought out solution. I changed my environment to start using http:// instead of going to localhost.

**Pull request closed without merging**